_posts/2019-10-3-PPAP.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Pupubird
```

Comment:
Please change your title to something unique, it is currently clashing with an article [here](https://github.com/sunwaytechclub/hacktoberfest-2019/blob/master/_posts/2019-10-1-Pupubird.md)